### PR TITLE
issue #30 - Remove filenames in merged files

### PR DESCRIPTION
**Pull request by piotrgolasz**
I made this as a setting configurable in config file.

Basically it gets a boolean value and passes it to Asset_Collection where based on this value compile() function displays the filenames or not

### CODE REVIEW

File: classes/Kohana/Asset/Collection.php [hunk 1]
```diff
@@ -38,6 +38,30 @@
...
 	protected $_last_modified = NULL;
+	
+	/**
+	 * @var bool show merged files paths
+	 */
+	protected $_show_paths = TRUE;
+	
+	/**
+	 * Setter / Getter for merged files path displaying
+	 * @author Piotr Gołasz <pgolasz@gmail.com>
```
Comment (hkdobrev) [inline classes/Kohana/Asset/Collection.php]: I'm sorry, but could you remove the `@author` annotation as it doesn't bring much value, such PHPDoc becomes stale and Git blame is better anyway. We need to remove it from other parts of the source code as well.

File: config/asset-merger.php [hunk 2]
```diff
@@ -7,6 +7,7 @@
 		Assets::JAVASCRIPT => DOCROOT.'js'.DIRECTORY_SEPARATOR,
 		Assets::STYLESHEET => DOCROOT.'css'.DIRECTORY_SEPARATOR,
 	),
+	'show_paths' => TRUE,
```
Comment (hkdobrev) [inline config/asset-merger.php]: Thanks for keeping the default to be the current one so we don't have _breaking_ changes.

File: classes/Kohana/Assets.php [hunk 3]
```diff
@@ -120,10 +126,24 @@ public function __construct($name = 'all')
...
+		else
+		{
+			$this->_showpaths = $show_paths;
+		}
+		
 		foreach (array_keys($load_paths) as $type)
 		{
 			// Add asset groups
 			$this->_groups[$type] = new Asset_Collection($type, $name);
+			//Pass displaying of merged files paths
```
Comment (hkdobrev) [inline classes/Kohana/Assets.php]: Could you put a space between the `//` and the words after it like the other comments?

File: classes/Kohana/Assets.php [hunk 4]
```diff
@@ -120,10 +126,24 @@ public function __construct($name = 'all')
 		if ( ! $load_paths OR ! is_array($load_paths) OR count(array_diff(array_keys($load_paths), array(Assets::JAVASCRIPT, Assets::STYLESHEET))))
 			throw new Kohana_Exception('You must configure load_paths for asset-merger, as array with keys Assets::JAVASCRIPT AND Assets::STYLESHEET, and values the actual load paths');
 		
+		$show_paths = Kohana::$config->load('asset-merger.show_paths');
+		
+		if(!is_bool($show_paths))
+		{
+			//default
```
Comment (hkdobrev) [inline classes/Kohana/Assets.php]: I don't think there's a need for this comment here.

File: classes/Kohana/Asset/Collection.php [hunk 5]
```diff
@@ -38,6 +38,30 @@
...
+	 * @param bool/null $path
+	 * @return bool
+	 */
+	public function display_paths($path = NULL)
+	{
+		if(is_null($path))
```
Comment (hkdobrev) [inline classes/Kohana/Asset/Collection.php]: Could you put a space between `if` and the opening parenthesis? Same thing a couple places below. Thanks!

**Comment by piotrgolasz** [comment]
@hkdobrev fixed code formatting accordingly.

**Comment by hkdobrev** [comment]
LGTM 👍

**Pull request merged**